osu.Game/OsuGame.cs

```diff
@@ -272,6 +272,24 @@ protected override void LoadComplete()
             Cursor.State = Visibility.Hidden;
         }
 
+        private Task asyncLoadStream;
+
+        private void loadComponentSingleFile<T>(T d, Action<T> add)
+            where T : Drawable
+        {
+            Schedule(() =>

```

Comment:
Does this need to be scheduled? I'd like to avoid scheduling async operations otherwise.